test/cppsuite/README.md

```diff
@@ -1 +1,132 @@
-# WiredTiger cppsuite
+<div style="text-align: justify">
+
+# Introduction
+The cppsuite is a C++ framework designed to help developers write multithreaded and highly configurable testing for WiredTiger. It is intended to make stress testing as well as scenario-driven testing easy to implement. The framework provides built-in [components](#components) that each offers a set of features to enhance the testing experience and [database operations](#database-operations) with a default implementation that can be overridden at the test level. Each test has a corresponding [configuration file](#test-configuration-file) that defines the workload.
+
+# Table of contents
+- [Introduction](#introduction)
+- [Database operations](#database-operations)
+  * [Populate](#populate)
+  * [Insert](#insert)
+  * [Update](#update)
+  * [Remove](#remove)
+  * [Read](#read)
+  * [Custom](#custom)
+  * [Checkpoint](#checkpoint)
+  * [Validate](#validate)
+- [Components](#components)
+  * [Workload Manager](#workload-manager)
+  * [Operation Tracker](#operation-tracker)
+  * [Timestamp Manager](#timestamp-manager)
+  * [Metrics Monitor](#metrics-monitor)
+- [Test configuration file](#test-configuration-file)
+- [How to execute a test](#how-to-execute-a-test)
+
+# Database operations
+The database operations and their default implementation are described in the subsections below. During a test, a user-defined number of threads can be associated to each of those operations. Currently, the framework only supports operations on keys and values of type string. The values are randomly generated using a uniform distribution.
```

Comment:
We jump right into the details of DB operations, but we haven't described what they do yet. Can we add an intro sentence?